common/presentation/src/main/kotlin/com/pandulapeter/myTodos/presentation/ui/Demo1.kt

```diff
@@ -20,15 +20,16 @@ import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.style.TextAlign
 import com.pandulapeter.myTodos.domain.GetTodoListUseCase
 import com.pandulapeter.myTodos.model.TodoList
 import com.pandulapeter.myTodos.presentation.resources.Dimension
 import com.pandulapeter.myTodos.presentation.resources.Text
 import com.pandulapeter.myTodos.presentation.utilities.get
 
 @Composable
-internal fun ColumnScope.Demo1(
-    getTodoList: GetTodoListUseCase = get()
+internal fun Demo1(
```

Comment:
I think we should avoid forcing a Composable to be used only as a child of a specific parent (Column)